docker/example/app/config.yaml

```diff
@@ -0,0 +1,24 @@
+credentials:
+  users:
+    guest:
+      roles: [ super ]
+
+groups:
+  group-001:
+    replicasets:
+      replicaset-001:
+        instances:
+          master:
+            iproto:
+              listen:
+                - uri: master:3301
+            database:
+              mode: rw
```

Comment:
Nit: I would use `replication.failover: manual` and `replicaset-001.leader: master`. This failover mode doesn't allow to configure two instances as masters. See https://github.com/tarantool/tarantool/blob/c56998fad6a7f2cda4252b5cfde1ccef5050fe19/doc/examples/config/replicaset_manual_failover.yaml.

BTW, I wouldn't use names like `master` and `replica-X`, because a replica exists to become a master some day.

Reply:
Fixed.

Reply:
I did the first part, so two instances cannot be configured as masters in my case, right? :) So one of them is `master` indeed, others are `replica`. Moreover, `master` and `replica` are very common terms that many people use.